modules/sttp/README.md

```diff
@@ -0,0 +1,35 @@
+# sttp module for PureConfig
```

Comment:
Please write this README using tut, since it ensures the code in this page will always be up-to-date. You can use `pureconfig-cats` ([modules/cats/src/main/tut/README.md](https://raw.githubusercontent.com/pureconfig/pureconfig/master/modules/cats/src/main/tut/README.md)) or `pureconfig-hadoop` ([modules/hadoop/src/main/tut/README.md](https://raw.githubusercontent.com/pureconfig/pureconfig/master/modules/hadoop/src/main/tut/README.md)) as examples.

Reply:
Done